README.md

```diff
@@ -120,6 +120,10 @@ and [Trello][]), plus [node.js API bindings][node], an [isomorphic
 [translating documentation][transifex] if you're interested in
 contributing!
 
+* **Peer Reviews**. Zulip is all about community and helping each other
+out. Check the [peer review stream][peer-review] on chat.zulip.org to help
+review PRs and give comments on other people's work. 
```

Comment:
Maybe add something like "everyone is welcome to participate, even those new to Zulip!  Even just checking out the code, manually testing it, and posting on whether or not it worked for you is valuable"?